config/lamba_edge.js

```diff
@@ -0,0 +1,22 @@
+// Node.js Lambda @ Edge
+// Adds security headers to static site hosted in CloudFront (postmortems.pagerduty.com)
```

Comment:
does it matter? probably not. it's a comment. but, could clean this up to be the current ops guide url